basics/installation/advanced/httpd.md

```diff
@@ -95,7 +94,7 @@ Don't forget to edit this configuration to make it works.
     # with mod_rewrite or mod_autoindex
     <FilesMatch \.php$>
         # SetHandler proxy:fcgi://127.0.0.1:9000
-        SetHandler proxy:unix:/var/run/php/php7.2-fpm.sock|fcgi://dummy
+        SetHandler proxy:unix:/var/run/php/php8.3-fpm.sock|fcgi://dummy
```

Comment:
```suggestion
        SetHandler proxy:unix:/var/run/php/php8.1-fpm.sock|fcgi://dummy
```
You are modifying PrestaShop documentation for version 8 which is compatible with php 8.1 maximum ;) (and PS version 9 is compatible with php 8.2 and php 8.3)